### ISSUE TITLE
Change Yolov11 Backbone

### ISSUE THREAD

**Issue by houdaa90** [houdaa90/Change-Yolov11-Backbone#1]
I want to change the backbone layers of yolov11 to EfficicentNetV2L. I followed some instructions on GitHub; i modified the yolo11.yaml located in `ultralytics\cfg\models\11`  this way 
backbone:
  - [0, 1, CustomEfficientNetV2L, []] 
  
and i add a special script to define the EfficicentNetV2L layers as follows

import torch
import torch.nn as nn
from torchvision.models import efficientnet_v2_l

class CustomEfficientNetV2L(nn.Module):
    """Custom EfficientNetV2L Backbone."""

    def __init__(self):
        super(CustomEfficientNetV2L, self).__init__()
        self.model = efficientnet_v2_l(weights=None)  # Load EfficientNetV2L without pretrained weights
        self.model.features[0] = nn.Conv2d(3, 64, kernel_size=3, stride=2, padding=1, bias=False)  # Adjust input layer

    def forward(self, x):
        return self.model.features(x)  # Return features for detection model

  then finally i modified the model.py located in `ultralytics\models\yolo` by defininig new class as follows 

  from ultralytics.models.efficientnet_v2l import CustomEfficientNetV2L 
class DetectionModelWithCustomBackbone(DetectionModel): 
    def __init__(self, cfg, weights, device, task=None, verbose=False):
        """Initialize detection model with a custom EfficientNetV2L backbone."""
        super().__init__(cfg=cfg, weights=weights, device=device, task=task, verbose=verbose)
        self.backbone = CustomEfficientNetV2L()  # Replace backbone with custom EfficientNetV2L
        # Print backbone details for verification
        if verbose:
            print("Custom Backbone Initialized: EfficientNetV2L")
            print(self.backbone)

    def forward(self, x):
        """Custom forward pass using the custom backbone."""
        features = self.backbone(x)  # Extract features from backbone
        neck_output = self.neck(features)  # Process through the neck module
        return self.head(neck_output)  # Output from head module
                
                    "detect": {
                "model": DetectionModelWithCustomBackbone,  
                "validator": yolo.detect.DetectionValidator,
                "predictor": yolo.detect.DetectionPredictor,
                "trainer": yolo.detect.DetectionTrainer,

When training, i can find no error but same results are produced like i'm not changing the backbone. I'm not sure if both results are similar or the backbone is not changed at all. 

So, if this is the right procedures? and how can i make sure and print that the backbone layers are changed?
